src/main/java/core/basesyntax/FruitShop.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class FruitShop {
+    private Map<String, StoreOperations> operationsMap;
+    private FileExecutor fileExecutor;
```

Comment:
FIleService will be more logical, cause you don't execute file, you read/write to file